pkg/karpenter/providers/helm/helm_test.go

```diff
@@ -196,7 +254,15 @@ var _ = Describe("HelmInstaller", func() {
 				}
 				actionConfig.KubeClient = fakeKube
 				installerUnderTest.ActionConfig = actionConfig
-				Expect(installerUnderTest.InstallChart(context.Background(), "karpenter", "karpenter/karpenter", "karpenter", "0.4.3", values)).To(MatchError(ContainSubstring("failed to install chart: failed to install CRD crds/karpenter.sh_provisioners.yaml: nope")))
+				Expect(installerUnderTest.InstallChart(
+					context.Background(),
+					"karpenter",
+					"karpenter/karpenter",
+					"karpenter",
+					"0.4.3",
+					true,
+					values,
+				)).To(MatchError(ContainSubstring("failed to install chart: failed to install CRD crds/karpenter.sh_provisioners.yaml: nope")))
```

Comment:
This reads a bit funny `Expect(myFuncCall()).To(MatchError`. Its not as clean as `Expect(err).To(MatchErr` or `Expect(funcCall).To(Succeed())`. worth splitting out the `err := ` outside of the expectation?

Reply:
I don't mind

Reply:
plz :smile:

Reply:
I mean, I don't mind doing it. :D :D :D Sorry.